### PR TITLE
Stats page

### PR DESCRIPTION
**Pull request by lovrobiljeskovic**
Add Stats page

- Add `react-table` and `highcharts` deps
- Add Stats page and data provider, with vMTA distribution chart and stakers table
- Add total stakers metric
- Add query for all user lockups
- Fix Github release link and version number in footer

### CODE REVIEW

File: src/context/DataProvider/DataProvider.tsx [hunk 1]
```diff
@@ -30,8 +34,12 @@ export const DataProvider: FC<{}> = ({ children }) => {
   );
 };
 
-export const useDataState = (): DataState => useContext(dataStateCtx);
+export const useDataState = (): DataState | undefined => useContext(dataStateCtx);
```
Comment (alsco77) [inline src/context/DataProvider/DataProvider.tsx:37]: Hmm. this doesn't seem right

**Review by alsco77 — COMMENTED** [review]
Just a few points

File: src/context/DataProvider/DataProvider.tsx [hunk 2]
```diff
@@ -3,24 +3,28 @@ import React, { createContext, FC, useContext, useMemo } from 'react';
 import { useTokensState } from './TokensProvider';
 import { RawData, DataState, IncentivisedVotingLockup } from './types';
 import { transformRawData } from './transformRawData';
-import { useUserLockupsSubscription } from './subscriptions';
+import { useUserLockupsSubscription, useAllUserLockupsSubscription } from './subscriptions';
 import { useAccount } from '../UserProvider';
 
-const dataStateCtx = createContext<DataState>({} as DataState);
+const dataStateCtx = createContext<DataState | undefined>(undefined);
```
[the remark this answers is not in the document]
Comment (lovrobiljeskovic) [reply src/context/DataProvider/DataProvider.tsx:9]: Well sometimes the queries do not immediately return results(takes time to process) so it is a way of handling the empty `DataState` I suppose. We can either do that or just have the `initialState` as an empty object casted to `DataState`. I tested both versions and they are working just fine so it's about preference at this point.

File: src/graphql/mstable/query.graphql [hunk 3]
```diff
@@ -65,3 +65,19 @@ query UserLockups($account: Bytes, $hasAccount: Boolean!) @api(name: mstable) {
     }
   }
 }
+
+query AllUserLockups($minLockTime: BigInt!) @api(name: mstable) {
+  userLockups(
+    where: { lockTime_gte: $minLockTime }
+    first: 1000
```
Comment (JamesLefrere) [inline src/graphql/mstable/query.graphql:72]: PS we know this won't do in the long term, but for now there are only 385 stakers

**Review by JamesLefrere — APPROVED** [review]
LGTM!

**Review by JamesLefrere — APPROVED** [review]

**Pull request merged**